elastic/kibana
# [Response Ops][Alerting] Write alert duration out as microseconds

Fixes https://github.com/elastic/kibana/issues/167298

## Summary

Framework alerts client was writing alert duration out as a nanoseconds string, which is how it is stored in the task state. However, the rule registry lifecycle executor is writing duration out as microsecond number. Aligning to write the same unit and format as the lifecycle executor. 

## To Verify:
- On `main`, create an ES query rule with active alerts that shows up on the O11y alerts table. Let it run and see that the duration shown on the table is very large and incorrect.
- Switch to this branch and let the rule run again. Verify that the duration on the alerts table is now correct.
